src/ParquetMsg.chpl

```diff
@@ -305,14 +305,24 @@ module ParquetMsg {
                                         dsetname, numelems,
                                         dtype, compressed,
                                         errMsg): int;
-    var filenames: [0..#A.targetLocales().size] string;
+    // var filenames: [0..#A.targetLocales().size] string;
```

Comment:
i noticed a bunch of `//` commented out lines in this file. i guess these are the old code. do we really need them still?